util/docker/travis_script.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+set -e
+
+DOCKER_DIR=$(dirname "$0")
+
+# WPT Dashboard code tests
+"${DOCKER_DIR}/exec.sh" pycodestyle --exclude=*_pb2.py .
+"${DOCKER_DIR}/exec.sh" python -m unittest discover -p '*_test.py'
+"${DOCKER_DIR}/exec.sh" go test -v ./...
+"${DOCKER_DIR}/exec.sh" golint -set_exit_status
```

Comment:
So currently we run the Travis builds in a matrix which allows them to run concurrently. I believe we can preserve this behavior by pre-building the docker container in the Travis run (which is already done in this PR it looks like) and then running each of the above commands as something like:

```
docker run --entrypoint "/bin/bash" wptd-testrun-jenkins pycodestyle --exclude=*_pb2.py
# or
docker run --entrypoint "/bin/bash" wptd-testrun-jenkins python -m unittest discover -p '*_test.py'
```

What do you think?

Reply:
That sounds good, but I'm not sure I know how to configure using a matrix. My understanding is that `docker run` is a blocking call, so simply doing what you describe inside `travis_script.sh` will not speed up test runs. Is that right?